packages/remix/package.json

```diff
@@ -38,22 +53,22 @@
     "tslib": "2.4.1"
   },
   "devDependencies": {
-    "@remix-run/react": "^1.7.6",
-    "@remix-run/server-runtime": "^1.7.6",
+    "@remix-run/react": "^2.0.0",
+    "@remix-run/server-runtime": "^2.0.0",
     "@types/cookie": "^0.5.0",
     "@types/node": "^16.11.55",
     "@types/react": "*",
     "@types/react-dom": "*",
     "typescript": "*"
   },
   "peerDependencies": {
-    "@remix-run/react": "^1.7.6",
-    "@remix-run/server-runtime": "^1.7.6",
-    "react": ">=16.8",
-    "react-dom": ">=16.8"
+    "@remix-run/react": "^2.0.0",
+    "@remix-run/server-runtime": "^2.0.0",
+    "react": ">=18.0.0",
+    "react-dom": ">=18.0.0"
```

Comment:
We can keep react as peerDeps because if we'd switch to tsup in the future then those would be marked as `external` automatically